src/pwquality.conf

```diff
@@ -61,3 +61,15 @@
 #
 # Path to the cracklib dictionaries. Default is to use the cracklib default.
 # dictpath =
+#
+# Prompt user at most N times before returning with error. The default is 1.
+retry = 3
+#
+# Enforces pwquality checks on the root user password.
+# The check is disabled if the value is 0.
+# enforce_for_root = 1
```

Comment:
Correctly commented out, but there should not be any '= 1' And the text should be "Enabled if the option is present.'

Reply:
Fixed